### PR TITLE
Add Persian language with code fa-IR

### PR DESCRIPTION
**Pull request by MahdiMajidzadeh**

### CODE REVIEW

File: src/locale/fa-IR/_lib/formatDistance/test.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
+// @flow
+/* eslint-env mocha */
+
+import assert from 'power-assert'
+import formatDistance from '.'
+
+describe('en-US locale > formatDistance', function () {
```
Comment (kossnocorp) [inline src/locale/fa-IR/_lib/formatDistance/test.js]: Please update the description

File: src/locale/fa-IR/_lib/formatDistance/test.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  context('with a future suffix', function () {
+    it('adds `in` to a string', function () {
+      var result = formatDistance('halfAMinute', null, {
+        addSuffix: true,
+        comparison: 1
+      })
+      assert(result === 'in half a minute')
+    })
+  })
+})
```
Comment (kossnocorp) [inline src/locale/fa-IR/_lib/formatDistance/test.js]: These tests aren't implemented. In the work in progress?

File: src/locale/fa-IR/_lib/localize/index.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+var quarterValues = {
+  narrow: ['۱', '۲', '۳', '۴'],
+  abbreviated: ['ف۱', 'ف۲', 'ف۳', 'ف۴'],
+  wide: ['فصل اول', 'فصل دوم', 'فصل سوم', 'فصل چهارم']
+}
+
+// Note: in English, the names of days of the week and months are capitalized.
+// If you are making a new locale based on this one, check if the same is true for the language you're working on.
+// Generally, formatted dates should look like they are in the middle of a sentence,
+// e.g. in Spanish language the weekdays and months should be in the lowercase.
```
Comment (kossnocorp) [inline src/locale/fa-IR/_lib/localize/index.js]: Please remove the comment as it's not relevant anymore.

File: src/locale/fa-IR/_lib/localize/index.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+  // If ordinal numbers depend on context, for example,
+  // if they are different for different grammatical genders,
+  // use `options.unit`:
+  //
+  //   var options = dirtyOptions || {}
+  //   var unit = String(options.unit)
+  //
+  // where `unit` can be 'year', 'quarter', 'month', 'week', 'date', 'dayOfYear',
+  // 'day', 'hour', 'minute', 'second'
```
Comment (kossnocorp) [inline src/locale/fa-IR/_lib/localize/index.js]: Please remove the comment, it's not relevant anymore.

File: src/locale/fa-IR/index.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * @type {Locale}
+ * @category Locales
+ * @summary English locale (United States).
+ * @language English
+ * @iso-639-2 eng
+ * @author Sasha Koss [@kossnocorp]{@link https://github.com/kossnocorp}
+ * @author Lesha Koss [@leshakoss]{@link https://github.com/leshakoss}
+ */
```
Comment (kossnocorp) [inline src/locale/fa-IR/index.js]: Please update the JSDoc.

File: src/locale/fa-IR/test.js [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+import assert from 'power-assert'
+import locale from '.'
+
+import format from '../../format'
+import formatDistance from '../../formatDistance'
+import formatDistanceStrict from '../../formatDistanceStrict'
+import formatRelative from '../../formatRelative'
+import parse from '../../parse'
+
+describe('en-US locale', function () {
```
Comment (kossnocorp) [inline src/locale/fa-IR/test.js]: Description

File: src/locale/fa-IR/test.js [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+    })
+
+    it('ordinal time', function () {
+      var dateString = '1st hour, 2nd minute, 3rd second'
+      var formatString = "ho 'hour', mo 'minute', so 'second'"
+      var result = parse(dateString, formatString, baseDate, {locale: locale})
+      assert.deepEqual(result, new Date(1986, 3 /* Apr */, 4, 1, 2, 3))
+    })
+  })
+})
```
Comment (kossnocorp) [inline src/locale/fa-IR/test.js]: Also in progress?

**Comment by anilanar** [comment]
@MahdiMajidzadeh Ping.

**Comment by kossnocorp** [comment]
Closed by #1113

**Pull request closed without merging**